code/backend/Admin.py

```diff
@@ -21,6 +18,7 @@
 
 logger = logging.getLogger(__name__)
 logger.debug("Starting admin app")
+env_helper: EnvHelper = EnvHelper()
```

Comment:
I would recommend adding a comment here about why this is here, otherwise it looks like an unused variable and could be removed in the future

Reply:
I was planning to do another round of testing without this. But did not want to block this PR just because of this change.

Reply:
Removed the unused variable. Looks like it was because of the type declaration issue